Add parallel_tool_calls setting

BENMFeng opened the pull request:
Add parallel_tool_calls setting for building llama-api-server to run openai/swarm agent by replacing OPENAI_BASE_URL and OPENAI_API_KEY

juntao commented:
Hello, I am a [code review agent](https://github.com/flows-network/github-pr-review/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

BENMFeng commented:
> Hi @BENMFeng Please check your editor settings, it seems like you are trying to modify the newline with some special characters.

Yes, sorry, it may be because my mac mini was broken, I changed the windows IDE, didn't pay attention to the settings, and introduced a lot of abnormal character sets.
